### PR TITLE
update ew-credentials

### PR DESCRIPTION
**Pull request by JGiter**

### CODE REVIEW

**Comment by jrhender** [comment]
@JGiter Please take a look at the "Build and test" errors

File: src/modules/claim/resolvers/credential.resolver.ts [hunk 1]
```diff
@@ -1,17 +1,31 @@
...
   constructor(private readonly claimService: ClaimService) {}
-
   async getCredential(
+    did: string,
+    namespace: string
+  ): Promise<
+    VerifiableCredential<RoleCredentialSubject> | RoleEIP191JWT | null
+  > {
+    const vc = await this.getVerifiableCredential(did, namespace);
+    const eip191Jwt = await this.getEIP191JWT(did, namespace);
```
Comment (Harasz) [inline src/modules/claim/resolvers/credential.resolver.ts:21]: Maybe we can optimize it a little bit. We need vc or jwt, so why fetch both?

File: src/modules/claim/resolvers/credential.resolver.ts [hunk 2]
```diff
@@ -21,15 +35,30 @@ export class RoleCredentialResolver implements CredentialResolver {
...
     const claim = await this.claimService.getByClaimType({
       subject,
       claimType,
     });
+    if (!claim) {
+      return null;
+    }
+
+    const { issuedToken } = claim;
+    const { claimData, signer, credentialStatus } = jsonwebtoken.decode(
```
Comment (Harasz) [inline src/modules/claim/resolvers/credential.resolver.ts:51]: What if claim is only on-chain, so issuedToken is undefined?

**Review by Harasz — CHANGES_REQUESTED** [review]

File: src/modules/claim/resolvers/credential.resolver.ts (hunk 1 above)
Comment (JGiter) [reply src/modules/claim/resolvers/credential.resolver.ts:21]: Yes, make sense

**Review by jrhender — APPROVED** [review]
This looks good to me

**Comment by JGiter** [comment]
Could some admin merge? I unable because of audit failure

**Comment by jrhender** [comment]
I think we also need @Harasz to review again as he requested changes

**Review by Harasz — APPROVED** [review]

**Pull request merged**